File: Codeforces/Ahmed Adel/Level 0/W2 Sheet/F.py
# // F. Towers Upgrade
# // Codeforces - Week 2 Sheet
# // https://codeforces.com/group/42BcTs2lav/contest/404038/problem/F
# // 2024-03-25 16:16:02
w = input().split()
a, b, c, x = int(w[0]), int(w[1]), int(w[2]), abs(int(w[0])-int(w[1]))
m = 10**9+7
lst = [a**b%m, b**a%m, c**x%m]
lst.sort()
print(lst[0], lst[1], lst[2])

File: Codeforces/Ahmed Adel/Level 0/W1 contest/F.cpp
# // F. Logic Case
# // Codeforces - Week#1 Contest
# // https://codeforces.com/group/42BcTs2lav/contest/405441/problem/F
# // 2024-03-24 19:58:43

w = input().split()
x = int(w[0])
y = int(w[1])
if x > y:   
	x, y = y, x
x -= 1
res = ((y*(y+1)//2) - (x*(x+1)//2)) - x-y-1
print(int(res))